components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -448,7 +448,6 @@ - (void)viewWillLayoutSubviews {
       contentViewFrame.size.height += topAreaInsetForHeader;
     }
   } else {
-    contentViewFrame.size.height = _contentVCPreferredContentSizeHeightCached;
```

Comment:
why was this removed

Reply:
That was a mistake, it has been added back.